spanner/ddl/ast.go

```diff
@@ -328,45 +328,55 @@ func NewSchema() Schema {
 	return make(map[string]CreateTable)
 }
 
+// Tables are ordered in alphabetical order with one exception: interleaved
+// tables appear after the definition of their parent table.
+func OrderTable(s Schema) []string {
+	var tableNames, sortedTableNames []string
+	for t := range s {
+		tableNames = append(tableNames, t)
+	}
+	sort.Strings(tableNames)
+	tableQueue := tableNames
+	printed := make(map[string]bool)
```

Comment:
nit: Since we are not actually printing them here, should we rename this? maybe picked or marked? We should also edit the comments below that say print.

Reply:
Thanks for pointing out , made the change